### DISCUSSION TITLE
Playfield border display is different from skin editor's playfield anchors

### DISCUSSION THREAD

**Discussion by n0Oo0Oo0b** [ppy/osu#28579] [Ideas] [answered]
The change that increased Standard's playfield border display size to account for circle radius in 2024.625.0-lazer did not carry over to the skin editor's playfield layer anchors. This means skin elements aligned to it would be on the 'actual' playfield size and not on the displayed playfield size:

| before | after |
|---|---|
| ![before](https://github.com/ppy/osu/assets/84215197/295260dd-493b-4572-954c-dfa3eb825a2c) | ![osu_2024-06-25_18-08-25](https://github.com/ppy/osu/assets/84215197/5eb6ff91-c2cb-4a48-a9ce-a17a36d5bf46) |

This can't be solved with a constant offset either since the size of the playfield border display depends on CS. Since the skin layer is also purely visual anyway, I think it makes sense to make it match the playfield border display.

**Comment by peppy** [comment]
I agree that it's probably best that the playfield skin elements track the dynamic size of the playfield border.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/28657
